docs/plugins/premium/install-premium-plugins.md

```
@@ -0,0 +1,100 @@
+```ini
+@tsedio:registry=https://npm.pkg.github.com
+//npm.pkg.github.com/:_authToken=GH_TOKEN
+```
+
+- For **pnpm**, this configuration is also valid.
+
+### Using Yarn v1
+
+Add or update a `.yarnrc` file:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)